src/Prism.Core/Common/ParametersBase.cs

```diff
@@ -58,6 +68,12 @@ protected ParametersBase(string query)
             }
         }
 
+        /// <summary>
+        /// Searches Parameter collection and returns value if Collection contains key.
+        /// Otherswise returns null.
+        /// </summary>
+        /// <param name="key">The key for the value to be returned</param>
+        /// <returns></returns>
```

Comment:
this is still empty... be sure all elements have a description

Reply:
I beleive i have finally got all the tags filled ... sorry